framework/packages/abstract-client/src/account.rs

```diff
@@ -521,6 +539,22 @@ impl<Chain: CwEnv> Account<Chain> {
         }
     }
 
+    /// Install an service on the account.
+    /// if `install_on_sub_account` is `true`, the application will be installed on new a sub-account. (default)
```

Comment:
We disabled the sub-account install by default.
```suggestion
    /// if `install_on_sub_account` is `true`, the application will be installed on new a sub-account.
```